source/minini.c

```diff
@@ -49,17 +51,57 @@ static int _minini_handler(void* user, const char* section, const char* name, co
     return 0;
 }
 
+void minini_open_config()
+{
+    // Check to see if there's a config file on the SD Card first
+    minini_config_file = fopen("sdmc:/minute/minute.ini", "r");
+
+    // If not found, make sure we let the code know the config is going towards the SLC
+    if(!minini_config_file)
+    {
+        minini_config_using_slc = true;
+    }
+
+    // Now check to see if a config file is found on the SLC. This finds a config file from where the fw.img file is usually located at
+    if(minini_config_using_slc)
+    {
+        minini_config_file = fopen("slc:/sys/hax/minute/minute.ini", "r");
```

Comment:
Instead of using a global variable, this function could return the file location, if found, or NULL if not found.

Reply:
agree